templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,12 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 34**
+
+* [`POST /messages`]: Add option `moved_msgs_visible_to_all_subs` to
```

Comment:
I think maybe clearer wording might be something like `add_messages_to_history`?

(We avoid using `msgs` as an abbreviation anywhere in our code or API, as a sidenote -- it's valuable to have a consistent naming policy, and that for Zulip is no abbreviations).

Reply:
Yes!